react-native/setup/add-cocoapods.mo.md

```diff
@@ -0,0 +1,37 @@
+# [MO] Add Cocoapods to your project (~10min)
+
+## Owner: [Yann Leflour](https://github.com/yleflour)
+
+## Prerequisites:
+
+- Have [bundler](http://bundler.io) installed
+- Have a *Gemfile* at the root of your project
+
+## Steps:
+
+### 1. Add cocoapods (~5min)
+
+- In your *Gemfile* add the line `cocoapods`
+- Run `bundle install`
+- Go into the iOS project folder (`cd ios`)
+  - Initialize pods with `bundle exec pod init`
+  - Add `pods` to *.gitignore*
```

Comment:
That is not a good ios practice. They version pods. No joke.

Reply:
This is shameful, not doing that until we face an issue